specification/spine-directory.yaml

```diff
@@ -180,30 +196,51 @@ components:
             - test
           example: "active"
         connectionType:
-          type: string
-        managingOrganization:
-          type: object
-          properties:
-            system:
-              type: string
-              example: "https://fhir.nhs.uk/Id/ods-organization-code"
-            value:
-              type: string
-              example: "R8008"
-        payloadType:
           type: object
           properties:
             system:
               type: string
-              example: "http://terminology.hl7.org/CodeSystem/endpoint-payload-type"
+              example: "http://terminology.hl7.org/CodeSystem/endpoint-connection-type"
             code:
               type: string
-              example: "any"
+              example: "hl7-fhir-msg"
             display:
               type: string
-              example: "any"
+              example: "HL7 FHIR Messaging"
+        managingOrganization:
+          type: object
+          properties:
+            identifier:
+              type: object
+              properties:
+                system:
+                  type: string
+                  example: "https://fhir.nhs.uk/Id/ods-organization-code"
+                value:
+                  type: string
+                  example: "R8008"
```

Comment:
shouldn't be 'YES'?

Reply:
using this example supplied by IOPs. 

https://github.com/NHSDigital/spine-implementation-guide/blob/master/Examples/SDS-Endpoint-Example.xml